From 868834d1c81a7b917583b11567cda01f7cee0051 Mon Sep 17 00:00:00 2001
From: Arne Coucheron <arco68@gmail.com>
Date: Sun, 21 Jan 2018 11:17:51 +0100
Subject: [PATCH 12/12] Fix double action performed when pressing menu key

Setting a custom menu key action will do both the custom action
and the default action at the same time. Comparing code with 14.1
we're missing the return here.

Change-Id: I2f188ec78fae5068ffa1a5c80a8afa5ee2f167b5
---
 services/core/java/com/android/server/policy/PhoneWindowManager.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/services/core/java/com/android/server/policy/PhoneWindowManager.java b/services/core/java/com/android/server/policy/PhoneWindowManager.java
index 16f284b..d666a1e 100644
--- a/services/core/java/com/android/server/policy/PhoneWindowManager.java
+++ b/services/core/java/com/android/server/policy/PhoneWindowManager.java
@@ -3924,6 +3924,7 @@ public class PhoneWindowManager implements WindowManagerPolicy {
                     performKeyAction(mMenuPressAction, event);
                 }
             }
+            return -1;
         } else if (keyCode == KeyEvent.KEYCODE_SEARCH) {
             if (down) {
                 if (repeatCount == 0) {
-- 
2.7.4

